airflow/www/templates/airflow/dag.html

```diff
@@ -33,7 +33,13 @@ <h3 class="pull-left">
         <span style='color:#AAA;'>SUBDAG: </span> <span> {{ dag.dag_id }}</span>
       {% else %}
         <input id="pause_resume" dag_id="{{ dag.dag_id }}" type="checkbox" {{ "checked" if not dag.is_paused else "" }} data-toggle="toggle" data-size="mini" method="post">
-        <span style='color:#AAA;'>DAG: </span> <span> {{ dag.dag_id }}</span> <small class="text-muted"> {{ dag.description }} </small>
+        <span style='color:#AAA;'>DAG: </span>
+        <span> {{ dag.dag_id }}
+          <a href="{{ url_for('airflow.trigger', dag_id=dag.dag_id) }}"
+          onclick="return confirmTriggerDag('{{ dag.safe_dag_id }}')">
+           <span class="glyphicon glyphicon-play-circle" aria-hidden="true" data-original-title="Trigger Dag"></span>
+          </a>
```

Comment:
Adds an extra 'trigger' execute Dag - so that we can manually trigger a dag from inside the view dashboard (like "Gantt" or "Task View") instead of having to go to the main dashboard to find the run dag again, to find the trigger link again.. (This is a feature in later Airflow versions)